test/src/org/omegat/core/segmentation/SRXTest.java

```diff
@@ -93,8 +91,8 @@ public void testSrxReaderDefault() {
         assertEquals(18, mapRuleList.size());
         for (MapRule mapRule : mapRuleList) {
             if (mapRule.getPattern().equals("JA.*")) {
-                assertEquals(LanguageCodes.JAPANESE_CODE, mapRule.getLanguageCode());
-                assertEquals(OStrings.getString(LanguageCodes.JAPANESE_KEY), mapRule.getLanguage());
+                assertEquals(LanguageCodes.JAPANESE_CODE, mapRule.getLanguage());
+                assertEquals(OStrings.getString(LanguageCodes.JAPANESE_KEY), mapRule.getLanguageName());
             }
```

Comment:
Check here, `MapRule#getLanguage` returns CODE, and `MapRule#getLanguageName` returns localized name.

All previous versions failed here in environment other than English.